### DISCUSSION TITLE
Asset Inventory: Asset Data Requirements - Asset deterioration potential and rate (if known) – T/N

### DISCUSSION THREAD

**Discussion by sreyac** [CriticalAssetManagement/CAMS#124] [General]
Requires further discussion - perhaps clubbed with estimated reminaing life?

**Comment by sreyac** [comment]
Not as a result of disaster - but was anyway less resilient and you discover that in the event of a disaster 

Nice to have - not MVP

**Comment by luke-feeney** [comment]
Mark Vessely @BGC - his idea and we should raise with him to see about suggestions 

Temporal scale - to give some sense of life of asset -> needs replacement in '2-4 years' 

Drop down to start but needs some work so should be built as a specific module
